### PR TITLE
Fixed breadcrumb

### PR DESCRIPTION
**Pull request by loic294**
# Description

**Make sure the PR is against the `develop` branch (see [Contributing](https://github.com/nccgroup/ScoutSuite/blob/master/CONTRIBUTING.md)).**

Please include a summary of the change(s) and which issue(s) it addresses. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change

Select the relevant option(s):

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (optional)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ScoutSuite/frontend/src/components/Breadcrumb/Provider/index.js [hunk 1]
```diff
@@ -3,11 +3,13 @@ import { useAPI } from '../../../api/useAPI';
 import ChevronRightIcon from '@material-ui/icons/ChevronRight';
 
 import AWSLogo from './Logos/aws.png';
-import { Link, useParams } from 'react-router-dom';
+import GCPLogo from './Logos/gcp.png';
+import AzureLogo from './Logos/azure.png';
+import { Link, useParams } from '@reach/router';
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/components/Breadcrumb/Provider/index.js:8]: hadn't you changed it for `react-router-dom`?

File: ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+
+import { useAPI } from '../../../api/useAPI';
+import PropTypes from 'prop-types';
+import { Link, useParams, useLocation } from '@reach/router';
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js:6]: same here

**Review by SamuelMeilleur — CHANGES_REQUESTED** [review]

File: ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  const params = useParams();
+
+  if (loading) return null;
+
+  let resourceName = '';
+
+  for (let category of categories) {
+    for (let service of category.services) {
+      for (let res of service.resources) {
+        if (res.id === params.resource) resourceName = res.name;
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js:27]: using `Array.find` would be better here, something like: `resourceName = service.resources.find(res => res.id === params.resource).name;`

**Review by SamuelMeilleur — CHANGES_REQUESTED** [review]

File: ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js (hunk 3 above)
Comment (SamuelMeilleur) [reply ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js:27]: it's slightly more complicated than I thought to make it work, keep the one you prefer.
image

Comment (loic294) [reply ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js]: Found a much cleaner way of doing it 🙂
```js
const services = flatten(categories.map(c => c.services));
const resources = flatten(services.map(s => s.resources));
const resource = resources.find((res) => res.id === params.resource);
```

Comment (SamuelMeilleur) [reply ScoutSuite/frontend/src/components/Breadcrumb/Resources/index.js]: i like it, much better

**Review by SamuelMeilleur — APPROVED** [review]

**Pull request merged**